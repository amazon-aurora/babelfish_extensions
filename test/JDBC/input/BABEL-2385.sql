select cast(databasepropertyex('master','collation') as varchar(50))
go