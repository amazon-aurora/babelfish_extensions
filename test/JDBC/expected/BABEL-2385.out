select cast(databasepropertyex('master','collation') as varchar(50))
go
~~START~~
varchar
sql_latin1_general_cp1_ci_as
~~END~~

